index.html

```diff
@@ -26,24 +26,26 @@ <h1>RKI Formulare</h1>
       let cameras;
       let cameralabels = [];
       let j = 0;
-      
+      const html5QrCode = new Html5Qrcode("reader");
       function openCamera() {
         Html5Qrcode.getCameras().then(devices => {
           /**
            * devices would be an array of objects of type:
            * { id: "id", label: "label" }
            */
           if (devices && devices.length) {
+            let l = 0;
             cameraId = devices[j];
             cameras = devices;
             for(let i = 0; i < cameras.length; i++){
               cameralabels[i] = cameras[i].label;
+              l++;
             }
+            cameralabels[l] = "-----Kamera Wechseln-----";
```

Comment:
wenn du einen String häufiger im Code benutzt, solltest du den in einer const-Variable speichern